### DISCUSSION TITLE
Sunbird Rc- Configuring Schemas

### DISCUSSION THREAD

**Discussion by nagendraprasady** [Sunbird-RC/community#600] [Questions (FAQs)]
https://docs.sunbirdrc.dev/developer-documentation/developer-setup
After the click on Here, we unable to find the schemas.
https://github.com/sunbird-rc/sunbird-rc-core/tree/main/tools/cli/src/templates/examples/student-teacher/schemas/
<img width="916" alt="image" src="https://user-images.githubusercontent.com/128679814/232413023-b08e80cd-ea4f-4f63-80f9-1c314e98325e.png">

**Comment by tejash-jl** [comment]
@nagendraprasady 
The links have been fixed now, please check.

**Reply by nagendraprasady** [reply]
I try to download docker compose file using this command Url is not working
$ curl https://raw.githubusercontent.com/sunbird-rc/sunbird-rc-core/main/tools/cli/src/templates/examples/student-teacher/docker-compose.yaml > docker-compose.yml

https://raw.githubusercontent.com/sunbird-rc/sunbird-rc-core/main/tools/cli/src/templates/examples/student-teacher/docker-compose.yaml

**Reply by nagendraprasady** [reply]
@tejash-jl Can I get some solution of this Query as I was struck in that part.

**Reply by tejash-jl** [reply]
@nagendraprasady check this link for steps to setup registry locally
https://docs.sunbirdrc.dev/developer-documentation/installation-guide
